### PR TITLE
fix(dev): upgrade env

### PR DESCRIPTION
**Pull request by bodinsamuel**
- Update packages
- Setup circleci
- Reup travis for the moment
- Upgrade node and yarn
- Upgrade eslint and --fix

image

### CODE REVIEW

File: .husky/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+_
```
Comment (Haroenv) [inline .husky/.gitignore:1]: should this be a .gitkeep file instead?

File: package.json [hunk 2]
```diff
@@ -69,7 +64,7 @@
     "eslint-plugin-jsdoc": "35.1.2",
     "eslint-plugin-prettier": "3.4.0",
     "eslint-plugin-react": "7.24.0",
-    "husky": "6.0.0",
+    "husky": "^6.0.0",
```
Comment (Haroenv) [inline package.json:67]: other dependencies are installed exactly (no caret)

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-12.10.0
+14.16.1
```
Comment (Haroenv) [inline .nvmrc:1]: we can remove --experimental-modules now

Comment (Haroenv) [inline .nvmrc:1]: why not migrated to v16?

File: .husky/.gitignore (hunk 1 above)
Comment (bodinsamuel) [reply .husky/.gitignore:1]: apparently not (this is the automated install from husky)

**Review by Haroenv — COMMENTED** [review]
👍

File: .nvmrc (hunk 3 above)
Comment (bodinsamuel) [reply .nvmrc:1]: it's not yet LTS, it's in october

**Pull request merged**